packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -142,6 +157,11 @@ export class ExecutionEngineHttp implements IExecutionEngine {
     this.rpc.emitter.on(JsonRpcHttpClientEvent.RESPONSE, () => {
       this.updateEngineState(getExecutionEngineState({targetState: ExecutionEngineState.ONLINE, oldState: this.state}));
     });
+
+    // Initial engine state is ONLINE. Need to explicitly call `getClientVersion` once on startup
+    this.getClientVersion(getLodestarClientVersion(this.opts)).catch((e) => {
+      this.logger.error("Unable to get client version", {caller: "ExecutionEngineHttp constructor"}, e);
```

Comment:
I am really not a fan of having the `caller` here, is there anywhere else in the code base we do this in logs? we should just rely on already existing stack trace since error is passed to logger here as well